modules/EvseV2G/connection.cpp

```diff
@@ -985,3 +1021,51 @@ int connection_start_servers(struct v2g_context* ctx) {
 
     return 0;
 }
+int create_udp_socket(const int udp_port) {
+    // find suitable addresses
+    addrinfo hints;
+    addrinfo *result, *result_pointer;
+    int udp_socket;
+
+    memset(&hints, 0, sizeof(addrinfo));
+    hints.ai_family = AF_UNSPEC;    // allow IPv4 or IPv6
+    hints.ai_socktype = SOCK_DGRAM; // Datagram socket
+    hints.ai_flags = AI_ALL;        // For IPv4 or IPv6
+    hints.ai_protocol = 0;          // OS selects best protocol (best practise)
+
+    // convert port to string
+    char port_str[6];
```

Comment:
I might be a bit picky here, but the function takes an int, so it could be also 100000, which won't fit into 5 chars.  So maybe do a validation or accept an `uint16_t`.  I would probably also prefer `std::to_string` instead of `snprintf`.

Reply:
True, that would be a bug. Maybe it would make sense to limit the function input to 16 bits instead as that's the ports max size